res/database/Default/database_news.xml

```diff
@@ -897,11 +897,11 @@ Her normal body size is expected to be reached in a week.</en>
 		<news id="32617459-a2eb-412a-87c9-0411314c0694" thread_id="24f462b9-bac5-499e-ae2d-780f225872b1" type="0" creator="">
 			<title>
 				<de>Automaut kommt</de>
-				<en>Car toll comes</en>
+				<en>Car toll coming</en>
 			</title>
 			<description>
-				<de>Nach schweizer Vorbild wird zum 1. Januar die Automaut eingeführt. Autofahrer waren empört, bis sie hörten, dass die Niederländer Deutschland umfahren werden.</de>
-				<en>Modeled on the Swiss approach, the car toll is introduced on January 1st. Motorists were outraged until they heard that the Dutch will bypass Germany.</en>
+				<de>Nach Schweizer Vorbild wird zum 1. Januar die Automaut eingeführt. Autofahrer waren empört, bis sie hörten, dass die Niederländer Deutschland umfahren werden.</de>
+				<en>Following the Swiss example, car tolls will be introduced by January 1st. Motorists were outraged until they heard that the Dutch will be bypassing Germany.</en>
```

Comment:
...or just 'on'.

But I guess I went with 'by' because the German also says 'zum', not 'am', and I think that has a similar room for interpretation that it could - perhaps - happen earlier.

So leaving as is.